packages/firestore/src/remote/connection.ts

```diff
@@ -92,6 +92,8 @@ export interface Connection {
    */
   readonly shouldResourcePathBeIncludedInRequest: boolean;
 
+  terminate(): void;
```

Comment:
Could you add a comment here? I know the name is pretty self explanatory, but perhaps warning what the implications are if you don't call this method.

Reply:
Added. Thanks for the review @DellaBitta